stickynotes/stickynotes_applet.c

```diff
@@ -31,595 +31,627 @@ StickyNotes *stickynotes = NULL;
+
+    width = g_settings_get_int (stickynotes->settings,
+                                "default-width");
+    width = MAX (width, 1);
+    height = g_settings_get_int (stickynotes->settings,
+                                 "default-height");
+    height = MAX (height, 1);
+
+    sys_color = g_settings_get_boolean (stickynotes->settings,
+                                        "use-system-color");
+    sys_font = g_settings_get_boolean (stickynotes->settings,
+                                       "use-system-font");
+    sticky = g_settings_get_boolean (stickynotes->settings,
+                                     "sticky");
+    force_default = g_settings_get_boolean (stickynotes->settings,
+                                            "force-default");
+    desktop_hide = g_settings_get_boolean (stickynotes->settings,
+                                           "desktop-hide");
+
+    font_str = g_settings_get_string (stickynotes->settings,
+                                      "default-font");
+    if (!font_str)
+        font_str = g_strdup ("Sans 10");
+
+    color_str = g_settings_get_string (stickynotes->settings,
+                                       "default-color");
+    if (!color_str)
+        color_str = g_strdup ("#ECF833");
+
+    font_color_str = g_settings_get_string (stickynotes->settings,
+                                            "default-font-color");
+    if (!font_color_str)
+        font_color_str = g_strdup ("#000000");
+
+    gdk_rgba_parse (&color, color_str);
+    gdk_rgba_parse (&font_color, font_color_str);
+
+    g_free (color_str);
+    g_free (font_color_str);
+
+    gtk_adjustment_set_value (stickynotes->w_prefs_width, width);
+    gtk_adjustment_set_value (stickynotes->w_prefs_height, height);
+    gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (stickynotes->w_prefs_sys_color),
+                                                     sys_color);
+    gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (stickynotes->w_prefs_sys_font),
+                                                     sys_font);
+    gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (stickynotes->w_prefs_sticky),
+                                                     sticky);
+    gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (stickynotes->w_prefs_force),
+                                                     force_default);
+    gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (stickynotes->w_prefs_desktop),
+                                                     desktop_hide);
+
+    gtk_color_chooser_set_rgba (GTK_COLOR_CHOOSER (stickynotes->w_prefs_color),
+                                                   &color);
+    gtk_color_chooser_set_rgba (GTK_COLOR_CHOOSER (stickynotes->w_prefs_font_color),
+                                                   &font_color);
+
+    gtk_font_button_set_font_name (GTK_FONT_BUTTON (stickynotes->w_prefs_font),
+                                                    font_str);
```

Comment:
the 2nd argument of the function is not aligned

Reply:
Opps, it was late :)